lib/puma/cluster.rb

```diff
@@ -340,7 +359,6 @@ def setup_signals
 
       Signal.trap "TTOU" do
         @options[:workers] -= 1 if @options[:workers] >= 2
-        @workers.last.term
         wakeup!
```

Comment:
So this ends up delaying the response to TTOU until the next run of check_workers, right?